Iniciante/2951/README.md

```diff
@@ -0,0 +1,49 @@
+``` c
+    char runa_atual, letras_runas[100];
+    int runas_existentes, amizade_necessaria, valor_amizade = 0, runas_recitadas, j, i, valor_runas[100];
+```
+
+Agora, leremos os valores de `runas_existentes` e `amizade_necessaria` através da função `scanf`.
+``` c
+    scanf("%d", &runas_existentes);
+    scanf("%d", &amizade_necessaria);
+```
+
+Na sequência, através de um loop `for` com limite o valor de `runas_existentes` (intervalo de `i`=0 até `i`=`runas_existentes`-1), leremos a letra que representa a runa em questão e seu valor através da função `scanf`. Sendo assim, alocaremos os dados de letra e valor para `letras_runas` e `valor_runas`, respectivamente. Um detalhe muito importante aqui é: dentro de `scanf` os parâmetros DEVEM ser `(" %c%d", &letras_runas[i], &valor_runas[i])`, esse espaço antes de "%c" faz com que a leitura de caractere da função ignore espaços em brancos, evitando problemas com as demais `scanf`.
+``` c
+    for (i=0; i<runas_existentes; i++) {
+        scanf(" %c%d", &letras_runas[i], &valor_runas[i]);
+    }
+```
+
+Dando continuidade, leremos o valor de `runas_recitadas` com mais uma função `scanf`. Ademais, dentro de um loop `for` com limite o valor de `runas_recitadas` (intervalo de `i`=0 até `i`=`runas_recitadas`), leremos as runas que foram recitadas, uma a uma, e atribuíremos à `runa_atual`. Desse modo, dentro do primeiro loop, conseguiremos através de outro loop `for`, com limite o valor de `runas_existentes` (intervalo de `j`=0 até `j`=`runas_existentes-1`), percorrer todos as letras que representam as runas que existem e compara-las com `runa_atual`. Logo, por meio de um estrutura condicional `if` comparamos as runas e atribuímos o valor delas à `valor_amizade`. 
+``` c
+    scanf("%d", &runas_recitadas);
+    for (i=0; i<runas_recitadas; i++) {
+        scanf(" %c", &runa_atual);
+        for (j=0; j<runas_existentes; j++) {
+            if (runa_atual == letras_runas[j]) {
+                valor_amizade += valor_runas[j];
+            }
+        }
+    }
+```
+Por fim, imprimimos o valor de amizade obtido com a função `printf`. Por meio de uma estrutura condicional `if` comparamos o valor da amizade que é preciso para derrotar Gollum com o valor obtido, se a amizade obtida for maior ou igual imprimimos "You shall pass!", caso não for, imprimimos "My precioooous".
+``` c
+    printf("%d\n", valor_amizade);
+    if (valor_amizade >= amizade_necessaria) {
+        printf("You shall pass!\n");
+    }
+    else {
+        printf("My precioooous\n");
+    }
+```
```

Comment:
Faltou o rodapé
```suggestion
``` c
    printf("%d\n", valor_amizade);
    if (valor_amizade >= amizade_necessaria) {
        printf("You shall pass!\n");
    }
    else {
        printf("My precioooous\n");
    }
```
 
Caso tenha alguma dúvida sobre este problema ou sobre a resolução, entre em contato com o PET-BCC pelo nosso
[Facebook](https://www.facebook.com/petbcc/),
[Instagram](https://www.instagram.com/petbcc.ufscar/)
ou nos mande um e-mail com o assunto "URI" para  petbcc.ufscar@gmail.com
```